app/services/list_tasks.rb

```diff
@@ -0,0 +1,16 @@
+class ListTasks < ListCollection
+
+  attr_defaultable :task_respository,  -> { Task }
```

Comment:
Probably don't need this repository declaration since you are using the project to find the tasks.